test/Datadog.Trace.Tests/Tagging/TagsListTests.cs

```diff
@@ -68,6 +68,39 @@ public void GetAll_When_MissingTags()
             ValidateTags(tags.GetAllTags(), values);
         }
 
+        [Fact]
```

Comment:
Changes from this file should be removed in a follow-up commit, as we decided to write our tests differently than OTel

Reply:
I have reverted these.